### PR TITLE
Trig plots

### PR DESCRIPTION
**Pull request by benkrikler**
Builds on top of @bundocka's trigPlots branch but swaps paths to be absolute paths, expanding relative ones during the `cmsl1t_batch` command's processing of the config so that the jobs on the batch system know where to look.  Files must of course be visible to the batch system, ie. on lxplus, make sure you're using afs for this to work.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -85,7 +85,11 @@ then
   # this gives you voms-proxy-*, xrdcp and other grid tools
   source /cvmfs/grid.cern.ch/etc/profile.d/setup-cvmfs-ui.sh
   # ROOT 6, voms-proxy-init and other things
+<<<<<<< HEAD:bin/env.sh
+  source /cvmfs/sft.cern.ch/lcg/views/LCG_94/x86_64-slc6-gcc62-opt/setup.sh
+=======
   source /cvmfs/sft.cern.ch/lcg/views/${LCG_VERSION}/${LCG_ARCH}/setup.sh
```
Comment (kreczko) [inline setup.sh]: this should be the correct one

**Comment by kreczko** [comment]
Is `run_lumi.csv` really needed?

File: setup.sh (hunk 1 above)
Comment (benkrikler) [reply setup.sh]: Oh whoops, sorry, I missed this conflict.

**Comment by benkrikler** [comment]
> Is `run_lumi.csv` really needed?

I think that's a question for @bundocka more than me.  I tried to make the edits I added on top of his branch minimal, just to get things working.

**Comment by benkrikler** [comment]
For speed, here are the changes this makes compared to the HEAD of Aaron's trigPlots branch: https://github.com/bundocka/cms-l1t-analysis/compare/trigPlots...benkrikler:trigPlots.

I've had to merge in the master branch due to conflicts (as you've spotted Luke) so they're not really edits that I made but they're differences to the version from Aaron.

File: setup.sh (hunk 1 above)
Comment (benkrikler) [reply setup.sh]: This should be solved now.

**Comment by kreczko** [comment]
I think the merge & squash created more chaos than it is worth - still cannot merge.
Do you mind if I make a new branch?

**Comment by benkrikler** [comment]
It looks fine to me: I can see the button "Merge pull request".  Maybe refresh the page?

**Comment by kreczko** [comment]
Done, but
image


I cannot even cherry-pick the first commit:
f999e81f6e64cb909e9916000e53309c52147d12


Well, if it looks OK for you, please merge

**Comment by benkrikler** [comment]
yes, but you're on rebase and merge.  Just try merging (which as you know I prefer anyway :p )

**Pull request merged**